pkg/cmd/start/root.go

```diff
@@ -57,17 +57,22 @@ var startCmd = &cobra.Command{
 		config, err := project.ConfigFromProjectPath("")
 		utils.CheckErr(err)
 
-		proj, err := project.FromConfig(config)
-		utils.CheckErr(err)
+		history := &history.History{
+			ProjectDir: config.Dir,
+		}
+
+		proj := &project.Project{
+			Name:    config.Name,
+			Dir:     config.Dir,
+			History: history,
+		}
 
 		dash, err := dashboard.New(proj, map[string]string{}, noBrowser || output.CI)
 		utils.CheckErr(err)
 
 		ls := run.NewLocalServices(&project.Project{
-			Name: "local",
-			History: &history.History{
-				ProjectDir: proj.Dir,
-			},
+			Name:    "local",
+			History: history,
```

Comment:
This constructs a new reference to project, can the existing reference be used?

Reply:
Not sure why the name is local here instead of the project name, any idea? That threw me off

Reply:
This looks like filler to accomodate for History without providing a History object directly (I haven't traced it but on its face it appears that way.)